### PR TITLE
Replace usage of evaluate with string parsing

### PR DESCRIPTION
**Pull request by dennis-behm**
For transforming some input properties to proper map objects, the evaluate method was used, which is a security concern. This is replaced with some simple string parsing.

### CODE REVIEW

File: Pipeline/CreateUCDComponentVersion/dbb-ucd-packaging.groovy [hunk 1]
```diff
@@ -474,7 +477,7 @@ def getContainerAttributes(String ds, Properties properties) {
 		def lastLevelQual = ds.tokenize('.').last()
 		if (properties.containerMapping) {
 			// obtain the deployType setting from the property
-			cMapping = evaluate(properties.containerMapping)
+			cMapping = parseJSONStringToMap(properties.containerMapping)
```
Comment (lburgess07) [inline Pipeline/CreateUCDComponentVersion/dbb-ucd-packaging.groovy:480]: `cMapping` is never properly defined as far as I can tell. Even if it works, I think this should read 'def cMapping = parse...'? Seems like that would be the best practice here

**Review by lburgess07 — CHANGES_REQUESTED** [review]
Just had one small comment to take a look at

File: Pipeline/CreateUCDComponentVersion/dbb-ucd-packaging.groovy (hunk 1 above)
Comment (dennis-behm) [reply Pipeline/CreateUCDComponentVersion/dbb-ucd-packaging.groovy]: @lburgess07 Good catch. Thanks for pointing that out. See new commit.

**Comment by dennis-behm** [comment]
My history was screwed up. I had to rebase to get the commit sign-offs in. Ready now.

**Review by lburgess07 — APPROVED** [review]
Looks good!

**Pull request merged**